src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/test_sql_commands.py

```diff
@@ -362,9 +382,8 @@ def body(self):
         self.cmd('sql elastic-pools db show-activity -g {} --server-name {} --elastic-pool-name {}'
                  .format(rg, self.sql_server_name, self.pool_name),
                  checks=[
-                     JMESPathCheck('[0].resourceGroup', rg),
-                     JMESPathCheck('[0].serverName', self.sql_server_name),
-                     JMESPathCheck('[0].currentElasticPoolName', self.pool_name)])
+                     JMESPathCheck('type(@)', 'array'),
+                     CustomCheck(self.custom_activity_checker)])
```

Comment:
My q. was directed at Travis if that wasn't clear.
So I think what Travis is suggesting is instead of `CustomCheck`, you can get the result of the command and then run your assertions on that. e.g.:
```
result = self.cmd('sql elastic-pools ...', checks=[JMESPathCheck(...)])
for activity in result:
   if activity['resourceGroup'] != expected_rg:
      raise AssertionError(...)
```
@tjprescott to clarify though.